doc/tutorials/distributed.rst

```diff
@@ -38,6 +38,8 @@ Once the topology has been set up, each node in the cluster is assigned a rank.
 
 The tracker is logically distinct from all nodes in the cluster, and does not actually perform any computation. Once it has finished setting up the topology, the tracker has finished its job. However, a user may choose to colocate the tracker with a master/worker node for simplicity. 
 
+The tracker initially communicates with all nodes in the cluster over SSH. This means that the tracker's SSH public key must be authorized by every node in the cluster. Once this initial communication has finished, all other inter-node communication happens over TLS.
```

Comment:
Instead of saying the tracker communicates with the other nodes over SSH, I think it might be clearer to rephrase this to something like: the tracker first logs into each node in the cluster via SSH to spin up the enclaves.
Then, merge this para with the next and say that the enclaves attest each other, and while doing so establish TLS connections with each other.

Reply:
Fixed